build_manpages/manpage.py

```diff
@@ -30,12 +31,14 @@
 MANPAGE_DATA_ATTRS = (
     "authors",  # just "author" in setup.cfg, can be specified multiple times
     "description",
-    "long_description",
+    "long_description",  # not in the manpage.py, we use parser.description
```

Comment:
@dmach, warning here ^ this was not used before when you introduced the option, and I prefer to not start using it...   Argparse object has it's description we can use.